apollo-router-scaffold/templates/plugin/src/plugins/{{snake_name}}.rs

```diff
@@ -1,26 +1,33 @@
+{{#if type_auth}}
+use std::ops::ControlFlow;
+
+use apollo_router::layers::ServiceBuilderExt;
+{{/if}}
+{{#if type_tracing}}
+use apollo_router::layers::ServiceBuilderExt;
+{{/if}}
 use apollo_router::plugin::Plugin;
 use apollo_router::plugin::PluginInit;
 use apollo_router::register_plugin;
-use apollo_router::services::supergraph;
 {{#if type_basic}}
-use apollo_router::services::router;
 use apollo_router::services::execution;
+{{/if}}
+{{#if type_basic}}
+use apollo_router::services::router;
 use apollo_router::services::subgraph;
 {{/if}}
+use apollo_router::services::supergraph;
+use schemars::JsonSchema;
+use serde::Deserialize;
+use tower::BoxError;
 {{#if type_auth}}
-use apollo_router::layers::ServiceBuilderExt;
-use std::ops::ControlFlow;
-use tower::ServiceExt;
 use tower::ServiceBuilder;
+use tower::ServiceExt;
 {{/if}}
 {{#if type_tracing}}
-use apollo_router::layers::ServiceBuilderExt;
-use tower::ServiceExt;
 use tower::ServiceBuilder;
+use tower::ServiceExt;
```

Comment:
Are these changes intentional? They look possibly made by a formatter that does understand template conditionnals

Reply:
yes, the formatter that understands template conditional was me :sweat_smile: 
The generated project that is now committed would be flagged by the lint step, the `ignore` option in `rustfmt.toml` is not stable yet, so I chose to format the project, then edit the template so it generates properly formatted code